src/main/java/uk/gov/ons/entities/Tokens.java

```diff
@@ -2,7 +2,12 @@
 
 import com.fasterxml.jackson.annotation.JsonAlias;
 
-public class Tokens {
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Getter;
+
+@Getter(value = AccessLevel.NONE)
```

Comment:
As youve had to override the getters to do the null check im not sure the @getter adds anything. bit of a bummer that null check.

Reply:
Agreed. Removed unnecessary code.